imap_processing/idex/idex_packet_parser.py

```diff
@@ -133,35 +155,318 @@ def __init__(self, header_packet):
+               triggering the low sample"""
+        # Trigger modes
+        trigger_dict["low_sample_trigger_mode"] = packet.data[
+            "IDX__TXHDRLSTRIGMODE"
+        ].raw_value
+        trigger_notes_dict[
+            "low_sample_trigger_mode"
+        ] = "Low sample trigger mode (0=disabled, 1=enabled)"
+        trigger_dict["tof_low_trigger_mode"] = packet.data[
+            "IDX__TXHDRLSTRIGMODE"
+        ].raw_value
+        trigger_notes_dict[
+            "tof_low_trigger_mode"
+        ] = "TOF Low trigger mode (0=disabled, 1=enabled)"
+        trigger_dict["tof_mid_trigger_mode"] = packet.data[
+            "IDX__TXHDRMGTRIGMODE"
+        ].raw_value
+        trigger_notes_dict[
+            "tof_mid_trigger_mode"
+        ] = "TOF Mid trigger mode (0=disabled, 1=enabled)"
+        trigger_dict["tof_high_trigger_mode"] = packet.data[
+            "IDX__TXHDRHGTRIGMODE"
+        ].raw_value
+        trigger_notes_dict[
+            "tof_high_trigger_mode"
+        ] = """TOF Mid trigger mode (0=disabled, 1=threshold mode, 2=single pulse
+               mode, 3=double pulse mode)"""
+
+        trigger_dict["detector_voltage"] = packet.data["IDX__TXHDRHVPSHKCH0"].raw_value
+        trigger_notes_dict[
+            "detector_voltage"
+        ] = "Last measurement in raw dN for processor board signal: Detector Voltage"
+        trigger_dict["sensor_voltage"] = packet.data["IDX__TXHDRHVPSHKCH1"].raw_value
+        trigger_notes_dict[
+            "sensor_voltage"
+        ] = "Last measurement in raw dN for processor board signal: Sensor Voltage"
+        trigger_dict["target_voltage"] = packet.data["IDX__TXHDRHVPSHKCH2"].raw_value
+        trigger_notes_dict[
+            "target_voltage"
+        ] = "Last measurement in raw dN for processor board signal: Target Voltage"
+        trigger_dict["reflectron_voltage"] = packet.data[
+            "IDX__TXHDRHVPSHKCH3"
+        ].raw_value
+        trigger_notes_dict[
+            "reflectron_voltage"
+        ] = "Last measurement in raw dN for processor board signal: Reflectron Voltage"
+        trigger_dict["rejection_voltage"] = packet.data["IDX__TXHDRHVPSHKCH4"].raw_value
+        trigger_notes_dict[
+            "rejection_voltage"
+        ] = "Last measurement in raw dN for processor board signal: Rejection Voltage"
+        trigger_dict["detector_current"] = packet.data["IDX__TXHDRHVPSHKCH5"].raw_value
+        trigger_notes_dict[
+            "detector_current"
+        ] = "Last measurement in raw dN for processor board signal: Detector Current"
+
+        return trigger_dict, trigger_notes_dict
 
     def _calc_impact_time(self, packet):
         """
-        This calculates the number of seconds since Jan 1 2012
+        This calculates the unix timestamp from the FPGA header information
+        We are given the number of seconds since Jan 1 2012, we need seconds since 1970.
+        Parameters
+        ----------
+            packet: The IDEX FPGA header packet
```

Comment:
```suggestion
            packet : `space_packet_parser.ParsedPacket`
                The IDEX FPGA header packet
```
or something like that to reference whatever you're passing in as the type.